### PR TITLE
Split travis

### PR DESCRIPTION
**Pull request by beckmani**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,29 +18,39 @@ branches:
...
-  - name: test
-    script:
-      - make genproto
-      - make test
-  - name: test-tidy-fmt
-    script:
-      - make genproto
-      - make test-tidy
-      - make test-fmt
+#  - stage: test
```
Comment (y0sher) [inline .travis.yml]: add the needed testing pipeline back

File: Makefile [hunk 2]
```diff
@@ -8,10 +8,15 @@ BIN_DIR = $(CURR_DIR)/build
 BIN_DIR_WIN = $(CUR_DIR_WIN)/build
 export GO111MODULE = on
 
-ifdef TRAVIS_BRANCH
-	BRANCH := $(TRAVIS_BRANCH)
-else
-	BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+#ifdef TRAVIS_BRANCH
```
Comment (y0sher) [inline Makefile]: remove old commented code

**Comment by beckmani** [comment]
Pusher,  don't review it yet.  It is still WIP

בתאריך יום ג׳, 18 ביוני 2019, 18:32, מאת rehs0y ‏<notifications@github.com>:

File: Makefile [hunk 3]
```diff
@@ -130,53 +135,87 @@ dockerbuild-go:
...
+		-e GOOGLE_APPLICATION_CREDENTIALS=./spacemesh.json \
+		-e CLIENT_DOCKER_IMAGE="spacemeshos/$(DOCKER_IMAGE_REPO):$(BRANCH)" \
+		-it go-spacemesh-python:$(BRANCH) pytest -s hare/test_hare.py --tc-file=hare/config.yaml --tc-format=yaml
+.PHONY: dockerrun-hare
+
+dockertest-hare: dockerbuild-test dockerrun-hare
+.PHONY: dockertest-hare
+
+dockerrun-sync:
+ifndef ES_PASSWD
```
Comment (y0sher) [inline Makefile:199]: maybe we can do this in a more general context instead of in every cmd

File: .travis.yml (hunk 1 above)
Comment (beckmani) [reply .travis.yml]: ok

File: Makefile (hunk 2 above)
Comment (beckmani) [reply Makefile]: ok

File: Makefile (hunk 3 above)
Comment (beckmani) [reply Makefile:199]: I do this in order to be able to run each command by it's own

**Review by noamnelke — APPROVED** [review]

**Pull request merged**